qdrant-landing/content/documentation/cloud/authentication.md

```diff
@@ -12,10 +12,10 @@ This page shows you how to use the Qdrant Cloud Console to create a custom API k
 The API key is only shown once after creation. If you lose it, you will need to create a new one. 
 However, we recommend rotating the keys from time to time. To create additional API keys do the following.
 
-1. Go to the **Access** section in the Dashboard.
-2. The **Access Management** list will display all available API keys.
+1. Go to the [Dashboard](https://qdrant.to/cloud).
+2. Select **Data Access Control** to display all available API keys.
 3. Click **Create** and choose a cluster name from the dropdown menu.
-> **Note:** You can create a key that provides access to multiple clusters. Simply check off which cluster in the dropdown
+> **Note:** You can create a key that provides access to multiple clusters. Select desired clusters in the dropdown box.
```

Comment:
Since you've suggested that users can set one API key for multiple clusters, I've made the sentence after that more consistent.